lib/src/components/NxTransferListHalf/__tests__/NxTransferListHalf.test.tsx

```diff
@@ -168,6 +168,54 @@ describe('NxTransferListHalf', function() {
     expect(items[1]).toHaveTextContent('baz');
   });
 
+  it('render from 0 to 3 items', function() {
```

Comment:
Add a comment here referencing the Jira ticket with the issue.  It isn't otherwise obvious why this test is especially necessary.

Reply:
> Add a comment here referencing the Jira ticket with the issue. It isn't otherwise obvious why this test is especially necessary.

done at ae5d0e0921ed84a67526a977b5ffb511b44ae9bc